.travis.yml

```diff
@@ -6,7 +6,11 @@ services:
   - docker
 
 env:
-    DOCKER_COMPOSE_VERSION: 1.10.1
+    global:
+        - DOCKER_COMPOSE_VERSION=1.10.1
+        # get all the branches referencing this commit
+        - IS_RELEASE=$(git ls-remote origin | grep "$TRAVIS_COMMIT\s\+refs/heads/release$"
+        - RELEASE_TAG=$(git ls-remote  --tags | awk '{print $2, $1}'  | awk -F"/" '{print $3 }' | sort -n -t. -k1,1 -k2,2 -k3,3 | grep -i $TRAVIS_COMMIT | cut -d\  -f 1)
```

Comment:
Question:
When I'm running locally:
```
MC-HQ-YMANOR:impact-api yotammanor$ export TRAVIS_COMMIT=51ba1c421db7d7d3a5bd411b91b64b910ec365bd
MC-HQ-YMANOR:impact-api yotammanor$ git ls-remote  --tags | awk '{print $2, $1}'  | awk -F"/" '{print $3 }' | sort -n -t. -k1,1 -k2,2 -k3,3 | grep -i $TRAVIS_COMMIT | cut -d\  -f 1
From https://github.com/masschallenge/impact-api.git
test-AC-5185-review-1
test-yotam
test-yotam-2017.1.4
```
Is this the expected output? (specifically - multi line, and the first line is **From ...git**)

Doing:
```
MC-HQ-YMANOR:impact-api yotammanor$ FOO=$(git ls-remote  --tags | awk '{print $2, $1}'  | awk -F"/" '{print $3 }' | sort -n -t. -k1,1 -k2,2 -k3,3 | grep -i $TRAVIS_COMMIT | cut -d\  -f 1)
MC-HQ-YMANOR:impact-api yotammanor$ echo $FOO
test-AC-5185-review-1 test-yotam test-yotam-2017.1.4
```

looks better (although how the omission of the first line happens is not clear to me). Is that correct?

Reply:
I came across this locally. If you try and pipe the output to a file or anywhere else it seems that the first line of output is not actual command output and is ignored so enclosing this in $() has the same effect. it shouldn't cause issues in the variable that gets set in travis.